app/scripts/migrations/105.test.ts

```diff
@@ -1,8 +1,8 @@
 import { v4 as uuid } from 'uuid';
 import { sha256FromString } from 'ethereumjs-util';
-import { InternalAccount } from '@metamask/keyring-internal-api';
 import { ETH_EOA_METHODS } from '../../../shared/constants/eth-methods';
 import { migrate } from './105';
+import type { Identity, InternalAccount } from './105';
```

Comment:
Why is it imported from the migration?

Reply:
Well.. given that the migration was done for a previous "state version", I thought that keeping the "old `InternalAccount` representation" for that migration was better than relying on the new one.

I honestly don't know what's the best strategy here.

So yes, the test import locally-defined type of the `InternalAccount` and `Identity` was duplicated in both the test and the migration files, so I just refactored slightly (I can revert this).